src/hooks/use-toast-messages.ts

```diff
@@ -0,0 +1,52 @@
+import { useCallback, useState } from 'preact/hooks';
+
+import type { ToastMessage } from '../components/feedback';
+
+export type ToastMessageData = Omit<ToastMessage, 'id'>;
+
+export type ToastMessageAppender = (toastMessageData: ToastMessageData) => void;
```

Comment:
What was the rationale for having this type alias, given that the signature is pretty simple?

Reply:
I don't remember 😅 

I don't really see a point for it, to be honest. We can get rid of it.

Reply:
Oh, I remembered. This type was being imported in one of via's components. This makes me realize that:

1. We need to make the types be exposed by the library's entry point.
2. I think we still don't need an alias for this type, as we can type as `ToastMessages['appendToastMessage']`. We have used this pattern in other places, and it gives a clear intention that those two types need to align.